sedaro/src/sedaro/block_class_client.py

```diff
@@ -50,6 +50,10 @@ def create(self, **fields) -> BlockClient:
         if is_empty(fields):
             raise ValueError(f'Must provide fields to create a {self.type}')
 
+        for kwarg in [ID, TYPE]:
+            if kwarg in fields:
```

Comment:
I'm strongly in favor of just removing these instead of erroring out.  This will require users to filter these out on their own which leads to lengthier, multi-line calls.

ex.

```py
del block.id
del block.type
Block.create(**block)

# or

Block.create(block.pop('id').pop('type')) # I suppose
```

I think this is unnecessarily restrictive.  If the user specifies create, I believe it is safe to omit the type and id fields.  We can say it the docs that they will be ignored.

Reply:
Done!